examples/lock-app/nrfconnect/CMakeLists.txt

```diff
@@ -23,6 +23,11 @@ include(${CHIP_ROOT}/config/nrfconnect/app/check-nrfconnect-version.cmake)
 
 # Load NCS/Zephyr build system
 set(CONF_FILE ${CHIP_ROOT}/config/nrfconnect/app/sample-defaults.conf prj.conf)
+# TODO: temporary fix to remove after solving static addressing problem on nrf5340
+if(${BOARD} STREQUAL "nrf5340dk_nrf5340_cpuapp")
```

Comment:
fyi: In CMake you don't need to use `${}` for most operators. `if (BOARD STREQUAL "nrf5340dk_nrf5340_cpuapp")` would be enough.